### ISSUE TITLE
The training set of the TSU dataset

### ISSUE THREAD

**Issue by Wyz2927** [dairui01/MS-TCT#4]
Hello, could you provide the training set of the TSU dataset? i.e. the block number of each stage, the learning rate and the batch size?

**Comment by dairui01** [comment]
You don't need to change any hyperparameters within the MS-TCT model. As TSU contains long videos but less dense co-occurring actions, you only need to change the batch size along with the learning rate. In our experiment, we utilize the cross-subject (CS) split of the dataset. We set batch size = 1, and learning rate = 0.0002. MS-TCT processes in a sliding window manner at inference time. To capture the long-term temporal relations among action instances in TSU, we set window size num_clip = 512. The obtained per frame mAP is around 34% and may vary to different GPU types (The result is validated and tested on 2 RTX6000 or 1 RTX8000).
